src/api-reference/request/v4.endpoints.travel-agency-resources.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: Travel Request v4 - Endpoints - Travel Agency Resources
+layout: reference
+---
+
+{% include prerelease.html %}
+
+If a new or existing project please use version [3.0](./v3.request.html) of this API.
+
+* [Travel Agency Resources](#travel-agency-resource)
+  * [Get the description of a travel agency office](#get-travel-agency-description-resource)
+
+## HTTP Status Codes
+
+To learn more about response HTTP status codes for this API see [Travel Request v4 - HTTP Status Codes](./v4.response-codes.html).
+
+# <a name="travel-agency-resource"></a>Travel Agency Resources
+
+Manage the configuration for travel agencies integrated with Concur Request.
+
+## <a name="get-travel-agency-description-resource"></a>Get the description of a travel agency office
+
+### Scopes
+
+`travelrequest.write` - Refer to [Scope Usage](./v4.get-started.html#scope-usage) for full details.
+
+### HTTP Request
+
+#### Parameters
+
+Name|Parameter Type|Data Type|Description
+---|---|---|---
+agencyUuid|path|string|**Required** The unique identifier of the Travel Agency
+
+##### URI Template
+
+```
+GET /v4/travelagencies/{agencyUuid}
+```
+
+#### Payload
+
+None
+
+### HTTP Response
+
+#### Payload
+
+Name|Type|Format|Description
```

Comment:
A little odd that this isn't on the Schemas page, to be consistent it should be there.